pkg/util/testing/wrappers.go

```diff
@@ -114,6 +114,21 @@ func (w *WorkloadWrapper) Admit(a *kueue.Admission) *WorkloadWrapper {
 	return w
 }
 
+func (w *WorkloadWrapper) Admitted(a bool) *WorkloadWrapper {
+	cond := metav1.Condition{
+		Type:               kueue.WorkloadAdmitted,
+		Status:             metav1.ConditionTrue,
+		LastTransitionTime: metav1.Now(),
+		Reason:             "AdmittedByTest",
```

Comment:
```suggestion
		Reason:             "ByTest",
```

since it could be false.